app/lib/ostatus/atom_serializer.rb

```diff
@@ -348,6 +348,10 @@ def add_namespaces(parent)
   def serialize_status_attributes(entry, status)
     append_element(entry, 'link', nil, rel: :alternate, type: 'application/activity+json', href: ActivityPub::TagManager.instance.uri_for(status)) if status.account.local?
 
+    if status.license_url
+      append_element(entry, 'link', rel: :license, type: 'application/rdf+xml', href: status.license_url)
```

Comment:
It should not say `type: 'application/rdf+xml'`.

I noticed that `LICENSE_URLS` are not for `application/rdf+xml`. For example, the RDF for CC BY is not `https://creativecommons.org/licenses/by/4.0/`, but `https://creativecommons.org/licenses/by/4.0/rdf`.
So either `LICENSE_URLS` or the `type` attribute must be changed. According to CC REL specification, removing the `type` attribute is better.